samples/amlv2_pid_symbol_detection_train/.vccode/launch.json

```diff
@@ -0,0 +1,177 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Creating ML Pipeline",
+            "type": "python",
+            "request": "launch",
+            "module": "automl_pipeline.pipeline",
+            "justMyCode": false,
+            "cwd": "${workspaceFolder}/src/app/"
+        },
+        {
+            "name": "Python: Create Azure ML Environment",
+            "type": "python",
+            "request": "launch",
+            "module": "automl_pipeline.mlops.create_environment",
+            "justMyCode": true,
+            "cwd": "${workspaceFolder}/src/app",
+            "args": [
+                "--environment-name",
+                "pid-environment",
+                "--environment-image",
+                "mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04",
+                "--environment-description",
+                "Environment belonging to Azure ML Python SDK v2 PID processing project"
+            ]
```

Comment:
Can you remove this run config? We removed this in the past engagement as we preferred to created anonymous environment.

Reply:
removed